editor/editor_node.cpp

```diff
@@ -7375,63 +7257,7 @@ EditorNode::EditorNode() {
 	raise_bottom_panel_item(AnimationPlayerEditor::get_singleton());
 
 	add_editor_plugin(VersionControlEditorPlugin::get_singleton());
-
-	// This list is alphabetized, and plugins that depend on Node2D are in their own section below.
-	add_editor_plugin(memnew(AnimationTreeEditorPlugin));
 	add_editor_plugin(memnew(AudioBusesEditorPlugin(audio_bus_editor)));
```

Comment:
Why wasn't AudioBusesEditorPlugin moved?

Reply:
Mostly because of the parameter. Also, the VS plugin (added above) was left there.